core/threaded/scheduler_LET.c

```diff
@@ -336,20 +337,16 @@ void lf_sched_init(
     }
 
     // Allocate array to hold information about what workers are in the workforce
-    worker_is_in_workforce = malloc(_lf_number_of_workers * sizeof(bool));
+    _lf_sched_worker_is_in_workforce = malloc(_lf_number_of_workers * sizeof(bool));
```

Comment:
The return value from malloc needs to be cast to bool* or this will fail with the CCpp target.